webmessaging/event.origin.sub.htm

```diff
@@ -9,7 +9,7 @@
 <div id=log></div>
 
 <div style="display:none">
-    <iframe width="70%" onload="PostMessageTest()" src="{{location[scheme]}}://{{domains[www]}}:{{location[port]}}/webmessaging/support/ChildWindowPostMessage.htm"></iframe>
+    <iframe width="70%" onload="PostMessageTest()" src="{{location[scheme]}}://{{domains[天気の良い日]}}:{{location[port]}}/webmessaging/support/ChildWindowPostMessage.htm"></iframe>
```

Comment:
Since at least Firefox was failing this, but perhaps passed the original test, can you keep the existing test and add add a new one?

(If we get to the point where regressions in wpt are taken more seriously, breaking existing tests will be less popular than adding new failing tests.)

Reply:
Firefox passes both variants here.

Reply:
OK, I guess https://github.com/w3c/web-platform-tests/pull/5941#issuecomment-301782745 is out of date, Firefox changed since then?

Reply:
Yeah. Firefox fixed its bug already.